### ISSUE TITLE
Doors not managed by the plugin

### ISSUE THREAD

**Issue by d-stephane** [brcDev/MyChunk#4]
Hello,
The following doors are not managed by the rights :
* BIRCH_DOOR_ITEM
* SPRUCE_DOOR_ITEM
* JUNGLE_DOOR_ITEM
* ACACIA_DOOR_ITEM
* DARK_OAK_DOOR_ITEM

**Comment by MrsWiccan** [comment]
Yup they still haven't updated it. This is an amazing plugin for simpler servers with few people.  They really need to get the ball rolling again.

**Comment by d-stephane** [comment]
it's possible this plugin is dead...
